.github/workflows/develop_commit_notification.yml

```diff
@@ -18,15 +23,13 @@ jobs:
           private_key: ${{ secrets.NOTIFICATIONS_TOKEN_GENERATOR_GH_APP_PRIVATE_KEY }}
       - name: Repository Dispatch
         uses: peter-evans/repository-dispatch@87c5425cae5ba8b5bc7da27674076c78588babf3
-        env:
-          VERSION: ${{ format('auto-{0}', github.sha) }}
         with:
           token: ${{ steps.generate_token.outputs.token }}
           repository: oppia/release-scripts
           event-type: develop-commit
           client-payload: '{
             "version": "${{ env.VERSION }}",
-            "app_name": "oppiaserver-backup-migration",
+            "app_name": "oppiatestserver",
```

Comment:
Ah, this is where we special-case the test server.

One question -- what was the previous code here doing, then? I didn't realize we did autodeployment to the backup server before...

Reply:
Yeah, I am not sure why it was that way. We were previously building RCs based on develop commits for the backup server. Though we never really used / deployed them.